src/web/src/components/SearchBar.tsx

```diff
@@ -95,7 +95,7 @@ const SearchBar = () => {
 
   const { filter, onFilterChange, onSubmitHandler } = useSearchValue();
 
-  const searchOptions = ['post', 'author'];
+  const searchOptions = ['post', 'author', 'date'];
```

Comment:
We have 2 dates in a post, so maybe we should keep the wording "published" somehow?